themes/aframe/source/js/common.js

```diff
@@ -175,9 +175,9 @@
   // DOCS.
   var anchorHeadingsSelector = 'h2[id], h3[id], h4[id], h5[id], h6[id]';
 
-  var contentDocs = $('.content--docs');
-  if (contentDocs) {
-    contentDocs.addEventListener('click', function (e) {
+  var contentStructured = $('.content--structured');
+  if (contentStructured) {
```

Comment:
maybe, enable if there are any `anchorHeadingsSelector` on the page. we just won't [style](https://github.com/aframevr/aframe-site-dev/pull/103/files#diff-0cc9234c5fe6e28e268a3d89fce00c9fR355) them.